pkg/constants/defaults.go

```diff
@@ -22,4 +26,7 @@ const (
 
 	// DefaultCLIEssentialsPluginGroupName  name of the essentials plugin group which is used to install essential plugins
 	DefaultCLIEssentialsPluginGroupName = "vmware-tanzucli/essentials"
+
+	// DefaultDigestExpirationThreshold is the default value for digest expiration threshold
+	DefaultDigestExpirationThreshold = 24 * time.Hour
```

Comment:
We need a more specific name I believe.  The digest expires after the 30 minutes TTL but the CLI won't notice until a command that needs the DB is run.  This 24 hours will force a digest refresh on any (almost) command of the CLI.  I like the naming you used for the env var below, so how about `DefaultPluginDBCacheRefreshThreshold`?

Reply:
Thanks , Renamed the variable